client/src/store/new-request-module.ts

```diff
@@ -135,6 +135,7 @@ export class NewRequestModule extends VuexModule {
   addressSuggestions: object | null = null
   allowAutoApprove: boolean = false
   analysisJSON: AnalysisJSONI | null = null
+  analyzePending: Boolean = false
```

Comment:
Don't use "Boolean".
https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html#number-string-boolean-symbol-and-object

Reply:
In fact, TS already knows this is a boolean, and in some environments the linter will throw an error about "redundant type", so I would just remove the type here.
(Rule of thumb: if you hover over the property and the proper type is displayed, it's good.)

Reply:
(I'm assuming you have some VS Code packages installed. Vetur is probably the main one :)

Reply:
haha sorry will update

Reply:
ahhh I only saw your first comment -- github just updated me with the other 2. I set it to boolean (I think this is okay because there are other values set to 'boolean' as well so it is consistent at least)

Reply:
Yeah, if we ever have to change this it will be a global effort. It's fine for now.